package.json

```diff
@@ -18,6 +18,7 @@
     "url": "http://github.com/deblockt/hal-rest-client/issues"
   },
   "dependencies": {
+    "@types/lodash": "^4.14.92",
```

Comment:
Is this lib very usefull? 
I'm trying to have a minimal dependency number.

Reply:
In my opinion it is. It adds the aspect of functional programming. It makes the code easier to read and to maintain. Since I'm a Java developer, this is something I miss sadly in TypeScript. I understand that you want to keep your footprint as small as possible and I can definitely rewrite the code to be imperative and if you insist, I'll do. But it'll make a tear run down my cheek. ;)